### PR TITLE
GRPC: add keepalive (120 sec)

### PR DESCRIPTION
**Pull request by AurelienGasser**
Add a keepalive (120 sec) to all the GRPC links﻿.

This prevents links from being closed by network software/equipment because of inactivity.

### CODE REVIEW

**Review by inalgnu — APPROVED** [review]

File: charts/substra-backend/values.yaml [hunk 1]
```diff
@@ -315,3 +315,7 @@ securityContext:
   fsGroup: 1001
   runAsUser: 1001
   runAsGroup: 1001
+
+grpc:
+  keepalive:
+    timeMs: 120000
```
Comment (AlexandrePicosson) [inline charts/substra-backend/values.yaml:321]: As this is only a backend setting and not enforced globally (currently only the backend use GRPC but we never know) shouldn't this be located under the `backend:` key ?

Comment (AurelienGasser) [reply charts/substra-backend/values.yaml:321]: That makes sense. 

Note: We have plans to separate the event app from the backend. If we do so, then the key would have to be moved again.

Comment (AlexandrePicosson) [reply charts/substra-backend/values.yaml:321]: Yes, but if you move the event app out of the backend you would also need to move it from here to an `eventApp:` key !

Comment (Kelvin-M) [reply charts/substra-backend/values.yaml:321]: I agree with Alexandre points.
We may first need to put this key under `backend`

Comment (AurelienGasser) [reply charts/substra-backend/values.yaml]: https://github.com/SubstraFoundation/substra-backend/pull/319/commits/072bfbf83d2f1a66d11d88ce036e4014077f6f1e

**Pull request merged**